src/udiskslinuxblock.c

```diff
@@ -1017,7 +1022,8 @@ udisks_linux_block_update (UDisksLinuxBlock       *block,
     {
       gchar *dm_uuid;
       dm_uuid = get_sysfs_attr (device->udev_device, "dm/uuid");
-      if (dm_uuid != NULL && g_str_has_prefix (dm_uuid, "CRYPT-LUKS"))
+      if (dm_uuid != NULL &&
+           (g_str_has_prefix (dm_uuid, "CRYPT-LUKS1") || g_str_has_prefix (dm_uuid, "CRYPT-TCRYPT")))
```

Comment:
Please change this back to `CRYPT-LUKS` only, so this works with both LUKS version 1 and 2. (Originally changed [here](https://github.com/storaged-project/udisks/commit/7d61552ed05b39bf4f63ac6edcbcccf2fa576a46))

Reply:
Oops, sorry, seems I was sloppy when I rebased on master. Fixed in e8741528.